spark-notebook.json

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "notebook",
+  "cpus": 0.25,
+  "mem": 2048.0,
+  "instances": 1,
+  "acceptedResourceRoles": [ "public" ],
```

Comment:
based on what jason wrote in [this doc](https://docs.google.com/document/d/1_JbMrctYJzCCg6qOCxcwU_5tv8zMPMNw0536guCqNdY/edit#), I think we need to use `slave_public` instead of `public`